packages/email/src/email-service.ts

```diff
@@ -0,0 +1,39 @@
+import sgMail from "@sendgrid/mail";
+
+// These functions are an abstraction over the sgMail module from sendgrid. The main
+// purpose is to throw runtime errors for blank api keys/from emails so we are alerted of
+// the issue ahead of time.
+
+function setApiKey(apiKey: string) {
+  if (apiKey == "") {
+    throw new TypeError("Invalid api key: Expected a non-empty string.");
+  }
+
+  sgMail.setApiKey(apiKey);
+}
+
+interface EmailMessage {
+  to: string;
+  from: string;
+  subject: string;
+  html: string;
+}
+
+async function send(msg: EmailMessage): Promise<boolean> {
+  if (msg.from == "") {
+    throw new TypeError("Invalid from email: Expected a non-empty string.");
+  }
+
+  try {
```

Comment:
It might be worth throwing the error directly, letting the consumer of the service deal with errors

just repalce the whole try catch with `return await sgMail.send(msg);`